sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/api/Microsoft.Azure.WebJobs.Extensions.ServiceBus.netstandard2.0.cs

```diff
@@ -111,6 +111,25 @@ public partial class SessionMessageProcessor
         protected internal virtual System.Threading.Tasks.Task CompleteProcessingMessageAsync(Microsoft.Azure.WebJobs.ServiceBus.ServiceBusSessionMessageActions actions, Azure.Messaging.ServiceBus.ServiceBusReceivedMessage message, Microsoft.Azure.WebJobs.Host.Executors.FunctionResult result, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
 }
+namespace Microsoft.Azure.WebJobs.ServiceBus.Listeners
+{
+    public partial class ServiceBusScaleMonitor : Microsoft.Azure.WebJobs.Host.Scale.IScaleMonitor, Microsoft.Azure.WebJobs.Host.Scale.IScaleMonitor<Microsoft.Azure.WebJobs.ServiceBus.Listeners.ServiceBusTriggerMetrics>
+    {
+        public ServiceBusScaleMonitor(string functionId, Microsoft.Azure.WebJobs.ServiceBus.ServiceBusEntityType serviceBusEntityType, string entityPath, string connection, Microsoft.Extensions.Logging.ILoggerFactory loggerFactory, Microsoft.Extensions.Configuration.IConfiguration configuration, Microsoft.Extensions.Azure.AzureComponentFactory componentFactory, Microsoft.Azure.WebJobs.ServiceBus.MessagingProvider messagingProvider, Microsoft.Extensions.Azure.AzureEventSourceLogForwarder logForwarder, Microsoft.Extensions.Options.IOptions<Microsoft.Azure.WebJobs.ServiceBus.ServiceBusOptions> options) { }
+        Microsoft.Azure.WebJobs.Host.Scale.ScaleMonitorDescriptor Microsoft.Azure.WebJobs.Host.Scale.IScaleMonitor.Descriptor { get { throw null; } }
+        System.Threading.Tasks.Task<Microsoft.Azure.WebJobs.Host.Scale.ScaleMetrics> Microsoft.Azure.WebJobs.Host.Scale.IScaleMonitor.GetMetricsAsync() { throw null; }
+        Microsoft.Azure.WebJobs.Host.Scale.ScaleStatus Microsoft.Azure.WebJobs.Host.Scale.IScaleMonitor.GetScaleStatus(Microsoft.Azure.WebJobs.Host.Scale.ScaleStatusContext context) { throw null; }
+        System.Threading.Tasks.Task<Microsoft.Azure.WebJobs.ServiceBus.Listeners.ServiceBusTriggerMetrics> Microsoft.Azure.WebJobs.Host.Scale.IScaleMonitor<Microsoft.Azure.WebJobs.ServiceBus.Listeners.ServiceBusTriggerMetrics>.GetMetricsAsync() { throw null; }
+        Microsoft.Azure.WebJobs.Host.Scale.ScaleStatus Microsoft.Azure.WebJobs.Host.Scale.IScaleMonitor<Microsoft.Azure.WebJobs.ServiceBus.Listeners.ServiceBusTriggerMetrics>.GetScaleStatus(Microsoft.Azure.WebJobs.Host.Scale.ScaleStatusContext<Microsoft.Azure.WebJobs.ServiceBus.Listeners.ServiceBusTriggerMetrics> context) { throw null; }
+    }
+    public partial class ServiceBusTriggerMetrics : Microsoft.Azure.WebJobs.Host.Scale.ScaleMetrics
+    {
+        public ServiceBusTriggerMetrics() { }
+        public long MessageCount { get { throw null; } }
+        public int PartitionCount { get { throw null; } }
+        public System.TimeSpan QueueTime { get { throw null; } }
```

Comment:
We may want to think about a better name for this property. What do you think of TimeOnQueue?

Reply:
There would be some complexity in changing these names at this point, since this is a serialization contract. These values are persisted and serialized/deserialized by hosts (e.g. Functions Host) and passed back into the monitor via GetScaleStatus.

Reply:
Ok, let's see if we can get this approved with the current name.